### PR TITLE
[native] Enable TPC-H q12 and q14 for Presto-on-Spark

### PR DESCRIPTION
**Pull request by vermapratyush**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-native-execution/etc/config.properties [hunk 1]
```diff
@@ -2,4 +2,5 @@ discovery.uri=http://127.0.0.1:58215
 presto.version=testversion
 http-server.http.port=7777
 shutdown-onset-sec=1
-register-test-functions=true
\ No newline at end of file
+register-test-functions=true
+shuffle.name=local
```
Comment (mbasmanova) [inline presto-native-execution/etc/config.properties:6]: Unrelated change?

File: presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeTpchQueries.java [hunk 2]
```diff
@@ -54,14 +54,6 @@ public void testTpchQ8() {}
     @Ignore
     public void testTpchQ11() {}
 
-    @Override
-    @Ignore
-    public void testTpchQ12() {}
```
Comment (mbasmanova) [inline presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeTpchQueries.java:59]: Since this PR enables only 2 queries, let's update PR title and commit message:

```
[native] Enabled TPC-H q12 and q14 for Presto-on-Spark
```

File: presto-native-execution/etc/config.properties (hunk 1 above)
Comment (vermapratyush) [reply presto-native-execution/etc/config.properties:6]: Related actually.
We need shuffle.name set to a default value for the unit test to run. shuffle.name set as local seems like the best default value.

Comment (mbasmanova) [reply presto-native-execution/etc/config.properties:6]: No, this is not needed for unit tests to run. Please, remove this change.

**Comment by mbasmanova** [comment]
@vermapratyush Looks like the PR is not based on latest master. Would you rebase?

File: presto-native-execution/etc/config.properties (hunk 1 above)
Comment (vermapratyush) [reply presto-native-execution/etc/config.properties:6]: @mbasmanova when running in detached mode, this property is indeed needed. Otherwise native cpp runs without any shuffle client.

Comment (mbasmanova) [reply presto-native-execution/etc/config.properties:6]: Sure, but this should be a separate PR and it requires splitting Prestissimo and Presto-on-Spark servers into separate binaries with separate set of configs.

**Review by mbasmanova — APPROVED** [review]
@vermapratyush Thanks.
